flashinfer/gdn_kernels/experimental/gdn_fused_decode_specialized.py

```
@@ -0,0 +1,725 @@
+                    mixed_qkv,
+                    conv_weight,
+                    conv_bias,
+                    conv_state,
+                    A_log,
+                    dt_bias,
+                    scale,
+                    ssm_state,
+                    state_indices,
+                    out=out,
+                )
+            except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)